### PR TITLE
Added three interlinked Spanish terms

### PR DESCRIPTION
**Pull request by callumrollo**
## Author: 

- callumrollo

## Language: 

- español

## Terms defined:

- importar
- librería
- módulo

## Editor

@ian-flores

Hi maintainers. I would like to contribute to this project. I see many terms that are defined only in English. Spanish is not my first language, but I have made the best I attempt I can with my knowledge and available tools. if it is inappropriate for a non-native speaker to contribute, please delete this PR.

Best wishes
Callum

### CODE REVIEW

**Comment by callumrollo** [comment]
Updated after reading page https://github.com/Carpentries-ES/board/blob/master/Convenciones_Traduccion.md linked from Issue #35. This document advises that biblioteca should be used throughout The Carpentries, as librería is an anglicism

File: glossary.yml [hunk 1]
```diff
@@ -2290,6 +2297,11 @@
     term: "library"
     def: >
       A reusable software package, also often called a [module](#module).
+  es:
+    term: "biblioteca"
```
Comment (ian-flores) [inline glossary.yml:2301]: Hola, creo que deberia ser libreria en vez de biblioteca

File: glossary.yml [hunk 2]
```diff
@@ -2669,6 +2681,10 @@
     term: "module"
     def: >
       A reusable software package, also often called a [library](#library).
+  es:
+    term: "módulo"
+    def: >
+      un paquete de software reutilizable, también se llama una [biblioteca](#library).
```
Comment (ian-flores) [inline glossary.yml:2687]: El mismo comentario de arriba, creo que deberia ser libreria.

**Review by ian-flores — CHANGES_REQUESTED** [review]

File: glossary.yml (hunk 1 above)
Comment (callumrollo) [reply glossary.yml:2301]: Hola, nela primera commit, use libreria. Luego vi en [este documento Carpentries](https://github.com/Carpentries-ES/board/blob/master/Convenciones_Traduccion.md)  de convenciones de traduccion, que se prefiere biblioteca como "el uso de librería ha sido descartado por ser un anglicismo". No obstante, no soy experto. Lo cambiare

Comment (callumrollo) [reply glossary.yml]: Hecho en 4e6c799

File: glossary.yml (hunk 2 above)
Comment (callumrollo) [reply glossary.yml]: Hecho en 4e6c799

**Comment by callumrollo** [comment]
Estes errores de Lint viene del Master 3142544c0d33b713987c5611dde510ebfb00b26d

**Review by baileythegreen — CHANGES_REQUESTED** [review]
This is failing the linter check because two terms, `absolute error` and `agile development`, have two entries for the Spanish definition.

Absolute error: lines 97 and 125
Agile development: lines 380 and 398

Please remove the duplicate entries, taking into consideration which of the given definitions is more complete.

**Comment by callumrollo** [comment]
@baileythegreen these duplicate terms have come from Master commit [3142544](https://github.com/carpentries/glosario/commit/3142544c0d33b713987c5611dde510ebfb00b26d). I had to pull fro master to resolve conflicts to complete this PR. I'm not sure what the cleanest way to resolve this is. Maybe Master should be fixed first them I'll pull again?

**Comment by baileythegreen** [comment]
@callumrollo I see. I'm on it. I'll submit a PR to revert the merge that caused it and ask the Spanish editor to look again.

**Comment by baileythegreen** [comment]
@callumrollo The PR with the duplicate keys has now been reverted. If you pull from master again, they should be removed from your PR, as well. Then it ought to pass the linter check.

File: glossary.yml (hunk 1 above)
Comment (dpshelio) [reply glossary.yml]: Hemos usado bibliotecas como traducción de "libraries" en todas nuestras lecciones siguiendo [las convenciones que creamos para ello](https://github.com/Carpentries-ES/board/blob/master/Convenciones_Traduccion.md). También [la documentación traducida de Python](https://docs.python.org/es/3/library/) usa bibliotecas y en [esta introducción a R](https://cran.r-project.org/doc/contrib/R-intro-1.1.0-espanol.1.pdf). Librerías tiene un significado distinto en español, y aunque usado en muchos lugares con dicho significado optamos por mantenerlo con la traducción clásica.
Vease la entrada en la [wikipedia a esta ambigüedad](https://es.wikipedia.org/wiki/Biblioteca_(inform%C3%A1tica)).

File: glossary.yml [hunk 3]
```diff
@@ -3318,6 +3325,10 @@
     term: "library"
     def: >
       A reusable software [package](#package), also often called a [module](#module).
+  es:
+    term: "librería"
```
Comment (dpshelio) [inline glossary.yml:3329]: Vease el comentario "obsoleto" en el hilo https://github.com/carpentries/glosario/pull/103#discussion_r541185464

**Review by dpshelio — CHANGES_REQUESTED** [review]

**Review by baileythegreen — APPROVED** [review]

**Comment by callumrollo** [comment]
@baileythegreen this should now merge cleanly into master. I've never done such a convoluted PR though so please tell me if I've screwed it up!

@dpshelio gracias por la aclaración

**Comment by baileythegreen** [comment]
I'm just 'approving' the 'changes' that mean this now passes the linter, because I don't want GitHub to list my review as 'pending' when it isn't needed.

**Comment by callumrollo** [comment]
Thanks @baileythegreen. Is there any more action needed on my part? Master is advancing and I'd like to get this merged so I can move on other Issues. I think @ian-flores needs to approve the change to biblioteca >> libreria following @dpshelio?

**Comment by baileythegreen** [comment]
@callumrollo 

Nothing from me, unless you run into lint issues due to upstream additions. Just waiting on Ian.

**Comment by nicoguaro** [comment]
> @callumrollo
> 
> Nothing from me, unless you run into lint issues due to upstream additions. Just waiting on Ian.

I have been added as an editor. Let me know if  there is something that I can help with.

**Review by ian-flores — APPROVED** [review]

**Comment by ian-flores** [comment]
Just accepted and will merge now

**Pull request merged**